.changeset/curly-turtles-melt.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-home': patch
+---
+
+- export `ContentModal` from `@backstage/plugin-home` so people can use this in other scenarios.
```

Comment:
Can we change this to remove the bullet list and have it as two sentences? I'm just not sure how well this will look once folded into the changelog - https://github.com/backstage/backstage/blob/master/plugins/home/CHANGELOG.md

Reply:
If you move the modal to home-react then you can do two changesets. That would also solve this 👍